webpack.config.js

```diff
@@ -111,23 +111,13 @@ function getWebpackConfig( { extensionName = '' } = {}, argv ) {
 			chunkIds: isDevelopment ? 'named' : 'natural',
 			minimize: shouldMinify,
 			minimizer: [
-				new UglifyJsPlugin( {
+				new TerserPlugin( {
 					cache: 'docker' !== process.env.CONTAINER,
 					parallel: true,
 					sourceMap: Boolean( process.env.SOURCEMAP ),
-					uglifyOptions: {
-						compress: {
-							/**
-							 * Produces inconsistent results
-							 * Enable when the following is resolved:
-							 * https://github.com/mishoo/UglifyJS2/issues/3010
-							 */
-							collapse_vars: false,
-						},
-						mangle: {
-							safari10: true,
-						},
+					terserOptions: {
 						ecma: 5,
+						safari10: true,
```

Comment:
According to [docs](https://github.com/fabiosantoscode/terser#minify-options), this is `mangle: { safari10: true }, output: { safari10: true }`. Do we prefer just the mangle safari option?

Reply:
we should be setting both. I don't think the output option was available when we added the mangle setting, but maybe we just missed it.